### ISSUE TITLE
Add support for liquid template engine

### ISSUE THREAD

**Issue by BigWillie** [Glavin001/atom-beautify#1416]
Please could you add support for  HTML (Liquid)

"Atom Beautify could not determine a supported beautifier to handle this file with grammar "HTML (Liquid)" and extension "liquid". If you would like to request support for this file and its language, please create an issue for Atom Beautify at https://github.com/Glavin001/atom-beautify/issues"

I can sort of fudge it by setting the file type to HTML (Twig) (liquid and twig have similar syntax)

**Comment by prettydiff** [comment]
First, test you code in Pretty Diff at http://prettydiff.com/?m=beautify and see if it breaks your code.  Twig uses Pretty Diff at the beautifier.

If all is well then I would create a pull request with a liquid.coffee file modeled after the Twig file at https://github.com/Glavin001/atom-beautify/blob/master/src/languages/twig.coffee

Once that pull request is merged in all should be well.

**Comment by prettydiff** [comment]
This comment might also be helpful: https://github.com/Glavin001/atom-beautify/issues/261#issuecomment-268017339

**Comment by BigWillie** [comment]
Yep - it works in prettydiff - indeed, it presumed the liquid was .twig 👍

**Comment by rickydazla** [comment]
Jekyll also uses Liquid FWIW

**Comment by Glavin001** [comment]
Closing as duplicate of https://github.com/Glavin001/atom-beautify/issues/393 and reopening https://github.com/Glavin001/atom-beautify/issues/393
